assets/js/romo/ajax.js

```diff
@@ -1,26 +1,29 @@
 var RomoAjax = function(element) {
-  this.elem       = $(element);
-  this.targetElem = $(this.elem.data('romo-ajax-target'));
+  this.elem = element;
+
+  var targetElemSelector = Romo.data(this.elem, 'romo-ajax-target');
+  if(targetElemSelector !== undefined) {
+    this.targetElem = Romo.f(targetElemSelector)[0];
+  } else [
+    this.targetElem = undefined;
+  }
```

Comment:
@jcredding we discussed this, but I don't think you need to do this undefined check - the f/find meths should return an empty array if passed null/undefined on their own.